azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -74,14 +74,29 @@ def __del__(self):
         """
         self.amqp_svc_client.disconnect_sync()
 
-    def create_device_with_sas(self, device_id, primary_key, secondary_key, status, iot_edge=False):
+    def create_device_with_sas(
+        self,
+        device_id,
+        primary_key,
+        secondary_key,
+        status,
+        iot_edge=False,
+        status_reason=None,
+        device_scope=None,
+        parent_scopes=None,
+    ):
         """Creates a device identity on IoTHub using SAS authentication.
 
         :param str device_id: The name (Id) of the device.
         :param str primary_key: Primary authentication key.
         :param str secondary_key: Secondary authentication key.
         :param str status: Initital state of the created device.
             (Possible values: "enabled" or "disabled")
+        :param bool iot_edge: Whether or not the created device is an IoT Edge device. Default value: False
+        :param str status_reason: The reason for the device identity status. Default value: None
+        :param str device_scope: The scope of the device. Default value: None
```

Comment:
Do you want to copy the doc comments I crafted in .NET? My opinion was that the comments from swagger were insufficient for end-user guidance.

Reply:
Done, thanks!